medusa/tv/series.py

```diff
@@ -2095,9 +2095,11 @@ def to_json(self, detailed=True):
         data['config']['defaultEpisodeStatus'] = self.default_ep_status_name
         data['config']['aliases'] = self.aliases
         data['config']['release'] = NonEmptyDict()
-        data['config']['release']['blacklist'] = bw_list.blacklist
-        data['config']['release']['whitelist'] = bw_list.whitelist
-        data['config']['release']['allgroups'] = get_release_groups_for_anime(self.name)
+        # These are for now considered anime-only options, as they query anidb for available release groups.
+        if self.is_anime:
```

Comment:
Won't this make it so it's missing the key blacklist instead of just setting it to be an empty array?

Reply:
Wouldn't that represent the current situation? Only anime shows have bl/wl.